### PR TITLE
Added create_directory, create_directories, list_files, create_files, search sent emails in Outlook

### PR DESCRIPTION
**Pull request by CraigWhiteART**

### CODE REVIEW

File: tests/test_email_search.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import unittest
+import os
+import sys
+sys.path.append(os.path.abspath('scripts'))
+
+from outlook_mapi_functions import search_sent_emails
+
+search_sent_emails("welcome")
```
Comment (nponeccop) [inline tests/test_email_search.py:8]: These file endings are against pep8. Change accordingly by ensuring that you don't occasionally delete the final newline.

File: scripts/commands.py [hunk 2]
```diff
@@ -179,7 +205,6 @@ def browse_website(url, question):
 
     return result
 
-
```
Comment (nponeccop) [inline scripts/commands.py:182]: Rollback the whiteline corrections. We'll do them in a separate PR (which is already there, just not merged yet)

**Review by nponeccop — CHANGES_REQUESTED** [review]

**Comment by p-i-** [comment]
This is a mass message from the AutoGPT core team.
Our apologies for the ongoing delay in processing PRs.
This is because we are re-architecting the AutoGPT core!

For more details (and for infor on joining our Discord), please refer to:
  https://github.com/Significant-Gravitas/Auto-GPT/wiki/Architecting

**Comment by Pwuts** [comment]
* Outlook-related functionality could be in a plugin (or maybe it already is?
* Other added commands aren't necessary with the current set of commands available in `master`

Closing this PR, thanks for submitting though!

**Pull request closed without merging**